messagingpush/src/main/java/io/customer/messagingpush/CustomerIOCloudMessagingReceiver.kt

```diff
@@ -27,8 +26,6 @@ class CustomerIOCloudMessagingReceiver : BroadcastReceiver() {
         val extras = intent.extras
         // Ignore event if no data was received in extras
         if (extras == null || extras.isEmpty) return
-        // If CustomerIO instance isn't initialized, we cannot process the notification
-        val sdkInstance = context.getSDKInstanceOrNull() ?: return
```

Comment:
This is important because it changes how metrics work. Unfortunately, it is tough to test with native as it is mostly utilized in wrappers. But because EventBus replays event, delayed SDK initialization shouldn't be issue here IMO.

Reply:
We should be able to verify this with a test as well

Reply:
lets a todo here? for test

Reply:
Sure, will try adding test for it. Adding todo for now.